server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/method/EmailQueryMethod.scala

```diff
@@ -40,6 +41,10 @@ import reactor.core.scala.publisher.{SFlux, SMono}
 
 import scala.jdk.CollectionConverters._
 
+object EmailQueryMethod {
+
+}
```

Comment:
Not needed?

Reply:
That's from the hellish rebase, escaped me. Thanks :)